tests/scripts/all.sh

```diff
@@ -444,7 +444,9 @@ pre_parse_command_line () {
 
     if [ $all_except -eq 0 ]; then
         unsupported=0
+        set -f
```

Comment:
Suggestion: perhaps add something similar to the commit message as a comment here?